.github/workflows/run-code-quality-check.yml

```diff
@@ -0,0 +1,90 @@
+---
+# MegaLinter GitHub Action configuration file
+# More info at https://megalinter.io
+name: MegaLinter
+
+on:
+  # Trigger mega-linter at every push. Action will also be visible from Pull Requests to main
+  push: # Comment this line to trigger action only on pull-requests (not recommended if you don't pay for GH Actions)
```

Comment:
Not sure if it's worth running this on all commits if there's no PR 🤔.

Also, can we _not_ run if the commit is on `develop` or `main`? As it's assumed the PR was already approved and merged, so this run would be unnecessary.

Reply:
That is all up to you, and your preferences. 

I believe there is no problem to configure it in the way you like. Probably running mega-linter once again doesn't add too much value as you state, since it was checked already.